timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -167,14 +118,15 @@ def run(self):
                 'And an overview of all the discovered search terms:')
             story.add_view(view)
 
+        output_string += '\n Problematic rules:'
+        output_string += ''.join(problem_string)
```

Comment:
same here as before, no concat, but rather `output_strings = []` and `return '\n'.join(output_strings)`